tests/api_tests/conftest.py

```diff
@@ -155,3 +156,15 @@ def _delete_comment(get_auth_token, comment_id):
         )
 
     return _delete_comment
+
+
+@pytest.fixture
+def fact_of_the_day_api():
+    base_url = "https://greencity.greencity.cx.ua/fact-of-the-day/random"
+
+    class BaseApi:
+        def get_data(self, headers=None, query_params=None):
+            response = requests.get(base_url, headers=headers, params=query_params)
+            return response
+
+    return BaseApi()
```

Comment:
The class BaseApi, with HTTP requests functionality already defined, is located in api/base_api.py